### DISCUSSION TITLE
Add custom events after Server has been set up

### DISCUSSION THREAD

**Discussion by TheRefineryAU** [plausible/analytics#4922] [Self-Hosted Support]
Can you add custom events to an already existing plausible site? I don't want to have to delete it and set it up again.

**Comment by ruslandoga** [comment]
👋 

It should be possible, yes.

You can add new custom properties at https://your.plausible.io/your.site/settings/properties (adapted to your domain / site)

**Reply by ruslandoga** [reply]
And you can get the script URL at https://your.plausible.io/your.site/installation?flow=review

<sub>That route should be linked from https://your.plausible.io/your.site/settings/general</sub>

**Comment by dawid-czarnecki** [comment]
I'm having the same problem. @ruslandoga The links you mentioned don't work for me. Both show me the dashboard with some data in it.
In the documentation I keep seeing "Site Installation" in General tab of site settings but my general tab contains these sections:
- Site domain
- Site timezone
- JavaScript snippet
- Data Import from Google Analytics

I've checked other tabs as well. Any idea where I can find these "Site settings"?

**Reply by dawid-czarnecki** [reply]
My setup is on docker: `plausible/analytics:latest`

**Reply by ruslandoga** [reply]
👋 @dawid-czarnecki 

Your image might be outdated. We switched away from DockerHub some time ago. The latest images are on `ghcr.io` now. Sorry for the confusion.

**Reply by ruslandoga** [reply]
`ghcr.io/plausible/community-edition:v3.0.1` is the current latest and greatest.

**Reply by ruslandoga** [reply]
More info:
- https://github.com/plausible/community-edition
- https://github.com/plausible/analytics/releases

**Reply by dawid-czarnecki** [reply]
My bad, I missed it. It works now, thanks!
